src/server/pfs/server/commit_store.go

```diff
@@ -0,0 +1,145 @@
+package server
+
+import (
+	"context"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/pachyderm/pachyderm/src/client/pfs"
+	"github.com/pachyderm/pachyderm/src/server/pkg/storage/fileset"
+	"github.com/pachyderm/pachyderm/src/server/pkg/storage/track"
+)
+
+type commitStore interface {
+	AddFileset(ctx context.Context, commit *pfs.Commit, filesetID fileset.ID) error
+	GetFileset(ctx context.Context, commit *pfs.Commit) (filesetID *fileset.ID, err error)
+	SetFileset(ctx context.Context, commit *pfs.Commit, id fileset.ID) error
+	DropFilesets(ctx context.Context, commit *pfs.Commit) error
+}
+
+var _ commitStore = &postgresCommitStore{}
+
+type postgresCommitStore struct {
+	db *sqlx.DB
+	s  *fileset.Storage
+	tr track.Tracker
+}
+
+func newPostgresCommitStore(db *sqlx.DB, tr track.Tracker, s *fileset.Storage) *postgresCommitStore {
+	return &postgresCommitStore{
+		db: db,
+		s:  s,
+		tr: tr,
+	}
+}
+
+func (cs *postgresCommitStore) AddFileset(ctx context.Context, commit *pfs.Commit, id fileset.ID) error {
+	// clone to remove the ttl.
+	id2, err := cs.s.Clone(ctx, id, track.NoTTL)
+	if err != nil {
+		return err
+	}
+	var num int
```

Comment:
I'm assuming this is going to be used for something at some point?

Reply:
I think I was logging it. I will remove it.